cypress/fixtures/members.ts

```diff
@@ -104,6 +97,13 @@ export const MEMBERS: {
     enableSaveActions: true,
     isValidated: true,
   },
+} satisfies {
+  [name: string]: CompleteMember & {
+    nameValid?: boolean;
+    emailValid?: boolean;
+    passwordValid?: boolean;
+    password?: Password;
+  };
```

Comment:
It is useful to use a type here? Maybe not, I'm just asking.

Reply:
It is only defined here and not re-used, I think it is fine and the call sites get the typing from the infered values so I guess it is ok ?